conn.c

```diff
@@ -33,13 +33,13 @@ make_conn(int fd, char start_state, tube use, tube watch)
     job j;
     Conn *c;
 
-    c = new(Conn);
-    if (!c) return twarn("OOM"), NULL;
+    c = (Conn *)new(Conn);
```

Comment:
It compiles without an error.  Same thing for `testserv.c`.  Would adding `-DNULL='((void*)0)'` to the `CFLAGS` when sunos is detected be a better solution?